### PR TITLE
New reference page for "list clusters"

### PR DESCRIPTION
**Pull request by marians**

### CODE REVIEW

File: content/reference/gsctl/create-keypair.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ weight = 32
 
 The `gsctl create keypair` command creates a key pair and stores the related files in your certificate folder.
 
-If you intend to use the created key pair with the `kubectl` program, we recommend you to take a look at the [`create kubeconfig`](../create-kubeconfig/) command. It creates a new key pair and adds the according user, server, and context entries to your `kubectl` configuration.
+If you intend to use the created key pair with the `kubectl` program, we recommend you take a look at the [`create kubeconfig`](../create-kubeconfig/) command. It creates a new key pair and adds the according user, server, and context entries to your `kubectl` configuration.
```
Comment (oponder) [inline content/reference/gsctl/create-keypair.md]: I think it should either be:

"We recommend taking a look"

or:

"We recommend that you take a look"

Comment (oponder) [inline content/reference/gsctl/create-keypair.md]: the that is optional, but I think the above suggestions read better personally

Comment (marians) [inline content/reference/gsctl/create-keypair.md]: Hmm. I wrote that after @rossf7's suggestion in https://github.com/giantswarm/docs-content/pull/243#discussion_r122701158

Comment (oponder) [inline content/reference/gsctl/create-keypair.md]: ah, it's subjective after all.

**Comment by marians** [comment]
Thanks!

**Pull request merged**